src/cobalt_strike.yml

```
@@ -52,3 +52,7 @@
       vars:
         # Group ownership for the Cobalt Strike addons
         group: wheel
+    - name: Install libreoffice
```

I think this should reference the product name and not the package.

```suggestion
    - name: Install LibreOffice
```